packages/jest-cli/src/TestPathPatternPrompt.js

```diff
@@ -122,35 +81,34 @@ module.exports = class TestPathPatternPrompt {
           );
           return highlight(path, filePath, pattern, context.config.rootDir);
         })
-        .map((item, i) => {
-          if (i === index) {
-            this._prompt.setTypheadheadSelection(chalk.stripColor(item));
-            return chalk.black.bgYellow(chalk.stripColor(item));
-          }
-          return item;
-        })
-        .forEach(filePath =>
-          this._pipe.write(`\n  ${chalk.dim('\u203A')} ${filePath}`),
-        );
+        .map((item, i) => formatTypeaheadSelection(item, i, index, prompt))
+        .forEach(item => printTypeaheadItem(item, pipe));
 
       if (total > max) {
-        const more = total - max;
-        this._pipe.write(
-          // eslint-disable-next-line max-len
-          `\n  ${chalk.dim(`\u203A and ${more} more ${pluralizeFile(more)}`)}`,
-        );
+        printMore('file', pipe, total - max);
       }
     } else {
-      this._pipe.write(
-        // eslint-disable-next-line max-len
-        `\n\n ${chalk.italic.yellow('Start typing to filter by a filename regex pattern.')}`,
-      );
+      printStartTyping('filename', pipe);
```

Comment:
I see now that I'm mixing singular with plural in couple of places, need to fix it properly.